Userland/Services/Taskbar/TaskbarWindow.cpp

```diff
@@ -54,8 +65,44 @@ class TaskbarWidget final : public GUI::Widget {
                 static_cast<TaskbarButton*>(button)->update_taskbar_rect();
         });
     }
+
+    ErrorOr<void> create_context_menu();
+
+    RefPtr<GUI::Menu> m_context_menu;
 };
 
+ErrorOr<NonnullRefPtr<TaskbarWidget>> TaskbarWidget::create()
+{
+    auto widget = TaskbarWidget::construct();
+    TRY(widget->create_context_menu());
+    return widget;
+}
+
+ErrorOr<void> TaskbarWidget::create_context_menu()
+{
+    m_context_menu = GUI::Menu::construct();
+
+    auto show_desktop_action = GUI::Action::create("Show &Desktop", TRY(Gfx::Bitmap::load_from_file("/res/icons/16x16/desktop.png"sv)), [](auto&) {
+        GUI::ConnectionToWindowManagerServer::the().async_toggle_show_desktop();
+    });
+
+    auto open_settings_action = GUI::Action::create("&Settings", TRY(Gfx::Bitmap::load_from_file("/res/icons/16x16/settings.png"sv)), [this](auto&) {
+        GUI::Process::spawn_or_show_error(window(), "/bin/Settings"sv);
+    });
+
+    auto open_system_monitor_action = GUI::Action::create("&System Monitor", TRY(Gfx::Bitmap::load_from_file("/res/icons/16x16/app-system-monitor.png"sv)), [this](auto&) {
```

Comment:
I think this leaves Settings and System Monitor with the same shortcut key 's'?

Reply:
ohhh! so thats what the ``&`` is for! 🤣fixed it!